src/main/java/co/rsk/bitcoinj/script/Script.java

```diff
@@ -677,7 +677,12 @@ public boolean isPayToScriptHash() {
      * Returns whether this script matches the format used for multisig outputs: [n] [keys...] [m] CHECKMULTISIG
      */
     public boolean isSentToMultiSig() {
-        return !this.getRedeemScriptParser().getMultiSigType().equals(MultiSigType.NO_MULTISIG_TYPE);
+        if (isSentToAddress() || isSentToRawPubKey()) {
+            return false;
+        }
+        else {
+            return !this.getRedeemScriptParser().getMultiSigType().equals(MultiSigType.NO_MULTISIG_TYPE);
+        }
```

Comment:
Are we sure this change covers all cases?

What about something like
```
try {
     multisigType = this.getRedeemScriptParser().getMultiSigType();
} catch (ScriptException) {
  return false;
}
    return !multisigType.equals(MultiSigType.NO_MULTISIG_TYPE);
```
Not sure if that's better but I see it closer to the original logic. We can discuss it

Reply:
Are there tests for this method?

Reply:
i think you are right. The original logic just asked for the parser type, but we had the NoRedeemScriptParser to avoid throwing errors. But checking just for sentToAddress or sentToRawPubKey may not be enough, we could end up throwing errors by mistake.